x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -60,25 +60,26 @@ For example, using the `005010` version and the 850 transaction set, the schema
 
 If you're using one or more custom schemas, refer to xref:x12-edi-connector-config-topics.adoc[XML Schema and Trading Partner Configuration] for complete information on setting your schema locations.
 
-== About X12 Connector Message Structure
+== About X12 EDI Connector Message Structure
 
 This connector enables the reading or writing of X12 documents into (or from) the canonical EDI message structure.
 This structure is a hierarchy of Java Maps and Lists. You use either DataWeave or code to manipulate them.
 Each transaction has its structure as defined in the schemas.
 
 The message itself contains the following keys, some of which only apply to either the read operation or the write operation, as indicated:
 
+
 [%header%autowidth.spread]
 |===
 |Key name |Description
 |Delimiters |The delimiters used for the message. If used, this set is based on the last interchange processed by a read operation. If set for a write operation, this overrides the values in the module configuration. The characters in the string are interpreted based on position in the following order: data separator, sub-element separator, repetition separator, or `U` if none, segment terminator.
-|Errors (read only) |A list of errors that are associated with the input as a whole, or with interchanges with no valid transaction sets
-|FunctionalAcksGenerated (read only) |A list of `997` or `999` configured acknowledgments that are generated by the module during the read operation
+|Errors (read only) |A list of errors that are associated with the input as a whole, or with interchanges with no valid transaction sets.
+|FunctionalAcksGenerated (read only) |A list of `997` or `999` configured acknowledgments that are generated by the module during the read operation.
 |Group (write only) |Map of GS group header segment values used as defaults when writing groups
-|Interchange (write only) |Map of ISA interchange header segment values used as defaults when writing interchanges
-|InterchangeAcksGenerated (read only) |A list of TA1 acknowledgments that were generated by the module during the read operation
-|InterchangeAcksReceived (read only) |A list of TA1 acknowledgments that were received by the module during the read operation
-|InterchangeAcksToSend (write only) |A list of TA1 acknowledgments that are sent by the module during the write operation
+|Interchange (write only) |Map of ISA interchange header segment values used as defaults when writing interchanges.
```

Comment:
parallelism and modification placement (the map is not doing the writing):

```suggestion
|Interchange (write only) |A map of ISA interchange header segment values used as defaults during an interchange write operation.
```

Reply:
Fixed.